team_admin/kit_shipping.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: Kit Shipping
+---
+
+# Kit Shipping
+
+This page provides details of how to ship a Student Robotics kit. Please only ship a Student Robotics kit if instructed to do so by the [Kit Logistics Coordinator](mailto:logistics@studentrobotics.org). If you have a kit that you need to ship to us please follow the instructions in the [Packaging](#packaging) section. Once you have your kit packaged, or are looking to get a kit collected from us to be delivered to you, please follow the instructions in the [Arranging a Courier](#arranging-a-courier) section.
```

Comment:
I would encourage explicitly including the anchor on the target of any internal links so that we're not at the mercy of whatever system happens to be rendering the page.

Note that the style of anchors used here does not match that used elsewhere within our docs (I think we used CamelCase mostly).